changedetectionio/html_tools.py

```diff
@@ -1,7 +1,10 @@
 import json
+import re
+from typing import List
```

Comment:
new to me! interesting

Reply:
Python type hints (check out https://docs.python.org/3/library/typing.html) To be honest, it's a bit odd that I added this single one - using them across the whole project would be cool though, could make development easier as IDEs can leverage them and inform you if expected input is violated.